### PR TITLE
Issues/307 add influxdb

### PR DESCRIPTION
**Pull request by haarts**
Prerequisite for https://github.com/monetas/gotary/issues/307.

I also took the opportunity to clean up some things along the way.

cleaaaaan

### CODE REVIEW

File: golang-base/run_services.sh [hunk 1]
```diff
@@ -18,3 +18,13 @@ for i in $(seq 5); do
...
 done
+
+# Wait for InfluxDB. It is *utterly* inconsequential which URL we curl.
+for i in $(seq 50); do
+   curl 'http://0.0.0.0:8086/' &> /dev/null
+   if [ "$?" = 0 ]; then
+       echo "InfluxDB started"
+       break
+   fi
+   sleep 1
```
Comment (mhamrle) [inline golang-base/run_services.sh]: It could be nice to see progress. In postgres waiting loop I can see result of `pg_isready`.

**Comment by mhamrle** [comment]
:+1:

**Comment by toxeus** [comment]
can we squash some of the one-liner commits?

**Comment by haarts** [comment]
I don't think squashing is a good idea now. That obfuscates the reason why certain changes were made.

File: golang-base/run_services.sh (hunk 1 above)
Comment (haarts) [inline golang-base/run_services.sh]: Done.

**Comment by toxeus** [comment]
> That obfuscates the reason why certain changes were made.

That's not true because the reason is in the commit msg.

**Comment by toxeus** [comment]
test this please

**Comment by toxeus** [comment]
test this please

**Comment by toxeus** [comment]
Moved to Gerrit:
https://gerrit.monetas.io/#/c/2/
https://gerrit.monetas.io/#/c/4/

**Pull request closed without merging**